src/frontend/js/components/Tabs/_styles.scss

```diff
@@ -0,0 +1,106 @@
+// .tabs{
+//   $background-color: r-theme-val(dashboard, background-color);
+
+//   &__header{
+//     display:flex;
+//     border-bottom : 2px solid gray;
+//     padding: 0 rem-calc(16px) 0 rem-calc(16px);
+
+//     &__tab {
+//       padding: rem-calc(4px) rem-calc(4px) 0 rem-calc(4px);
+//       border-radius:rem-calc(8px) rem-calc(8px) 0 0;
+//       border: 2px solid gray;
+//       border-bottom : 0;
+//       flex:0 0 auto;
+
+//       margin-right: rem-calc(8px);
+//       &:last-child{
+//         margin-right: 0;
+//       }
+
+//       &--active {
+//         border-bottom: 2px solid $background-color;
+//         margin-bottom: -2px;
+//       }
+//     }
+
+//     margin-bottom: rem-calc(16px);
+//   }
+// }
+
+$button-tab-scheme: map-get(
+  map-get(map-get(map-get($themes, 'default'), 'components'), 'button'),
+  'tab'
+);
+$default-border-radius: map-get($button-tab-scheme, 'border-radius');
+$default-color: map-get($button-tab-scheme, 'default-color');
+$hover-color: map-get($button-tab-scheme, 'hover-color');
+$focus-color: map-get($button-tab-scheme, 'focus-color');
+$selected-color: map-get($button-tab-scheme, 'selected-color');
+$disabled-color: map-get($button-tab-scheme, 'disabled-color');
+
+$tabs-scheme: map-get(map-get(map-get($themes, 'default'), 'components'), 'tabs');
+.tabs {
+  &__header {
+    display: flex;
+    border-bottom: 2px solid map-get($tabs-scheme, 'border-bottom-color');
+
+    &__tab {
+      margin-bottom: rem-calc(-2px);
+      flex: 0 0 auto;
+
+      margin-right: rem-calc(16px);
+      &:last-child {
+        margin-right: 0;
+      }
+    }
+  }
+}
+
+.c__button {
+  $default-color: map-get($button-tab-scheme, 'default-color');
+
+  &--tab {
+    // avoid movement when hovering/selecting a tab.
+    border: 1px solid transparent;
+    border-bottom-width: 2px;
```

Comment:
```suggestion
    border-bottom: 2px solid transparent;
```

Reply:
you right it work, that was a residue from my previous design, this fix isn't needed anymore